src/lang/lang_types.ml

```diff
@@ -622,11 +621,13 @@ let rec ( <: ) a b =
                   try (* TODO use variance info *)
                       h1 <: h2
                   with Error (a, b) ->
+                    let bt = Printexc.get_raw_backtrace () in
                     let post = List.map (fun (v, _) -> (v, `Ellipsis)) t1 in
-                    raise
+                    Printexc.raise_with_backtrace
```

Comment:
Didn't know this one!

Reply:
Yeah, it's quite useful to keep all the backtraces.